packages/commonwealth/client/scripts/views/modals/WelcomeOnboardModal/steps/PreferencesStep/constants.ts

```diff
@@ -0,0 +1,18 @@
+// TODO: these should come from API
+export const interestTags = [
+  '🐒 NFTs',
```

Comment:
not sure how the api would look like, but it would be good to have some ids on each tag